clicache/test/CMakeLists.txt

```diff
@@ -32,32 +32,35 @@ add_library(${PROJECT_NAME} MODULE
   ${CONFIGURE_OUT_FILES}
 )
 
-set_target_properties(${PROJECT_NAME} PROPERTIES
+set_target_properties(Apache.Geode.Test PROPERTIES
   VS_GLOBAL_CLRSupport "true"
   VS_GLOBAL_KEYWORD "ManagedCProj"
   VS_GLOBAL_TestProjectType UnitTest
   VS_GLOBAL_PROJECT_TYPES "{3AC096D0-A1C2-E12C-1390-A8335801FDAB};{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}"
-  VS_GLOBAL_ROOTNAMESPACE ${PROJECT_NAME}
+  VS_GLOBAL_ROOTNAMESPACE Apache.Geode.Test
   VS_DOTNET_TARGET_FRAMEWORK_VERSION "v4.5.2"
   VS_DOTNET_REFERENCES "System;System.Xml;Microsoft.VisualStudio.QualityTools.UnitTestFramework"
 )
 
-target_include_directories(${PROJECT_NAME}
+target_include_directories(Apache.Geode.Test
   PRIVATE
   	$<TARGET_PROPERTY:apache-geode,SOURCE_DIR>
   	$<TARGET_PROPERTY:apache-geode,INCLUDE_DIRECTORIES>
   	$<TARGET_PROPERTY:Apache.Geode,SOURCE_DIR>
 )
 
-target_link_libraries(${PROJECT_NAME}
+target_link_libraries(Apache.Geode.Test
   PUBLIC
-    Apache.Geode
+    # Apache.Geode #- Causes include of .lib
```

Comment:
It's a good question. I left the comment there because the "normal" use case is that this would be here but because CMake doesn't really understand the a C++/CLI can create a .DLL without a .LIB file it creates a dependency that fails. It is not the normal way to specify library dependencies by using the target dependency function used later in the file. Someone that later came back to this file might rightly believe the file was in error and "correct" it only to wonder why it doesn't work.